keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -234,11 +226,11 @@ A `Bucket` is not deleted if it is bound to a `BucketRequest`.
 1. `allowedNamespaces`: a list of namespaces that are permitted to either create new buckets or to access existing buckets.
 1. `protocol`: protocol-specific field the application will use to access the backend storage.
      - `name`: supported protocols are: "s3", "gs", and "azureBlob".
-     - `version`: version being used by the backend storage.
-     - `azureBlob`: data required to target a provisioned azure container.
-     - `gs`: data required to target a provisioned GS bucket.
-     - `s3`: data required to target a provisioned S3 bucket.
-1. `parameters`: a copy of the BucketClass parameters.
+     - `version`: protocol version being used by the backend storage.
+     - `bucketName`: the name of the bucket created by the backend storage.
```

Comment:
And for brownfield is this the requested bucket name or empty?

Reply:
The use case for bucketName hasn't changed.  All protocols have a bucketName already.  We just stopped trying to have the name be specific to the protocol.